applications/crossbar/priv/couchdb/schemas/system_config.speech.google.json

```diff
@@ -8,19 +8,44 @@
             "description": "API key for Google Cloud Speech",
             "type": "string"
         },
+        "asr_enable_automatic_punctuation": {
+            "default": true,
+            "description": "speech google asr_enable_automatic_punctuation",
+            "type": "boolean"
+        },
         "asr_enable_word_time_offsets": {
             "description": "top result includes a list of words and the start and end time offsets",
             "type": "boolean"
         },
+        "asr_encoding": {
+            "default": "LINEAR16",
+            "description": "speech google asr_encoding",
+            "type": "string"
+        },
+        "asr_model": {
+            "default": "phone_call",
+            "description": "speech google asr_model",
+            "type": "string"
+        },
         "asr_profanity_filter": {
             "description": "server will attempt to filter out profanities, replacing all but the initial character in each filtered word with asterisk",
             "type": "boolean"
         },
+        "asr_sample_rate_hertz": {
+            "default": "16000",
+            "description": "speech google asr_sample_rate_hertz",
+            "type": "string"
+        },
         "asr_url": {
             "default": "https://speech.googleapis.com/v1/speech:recognize",
             "description": "Google Cloud Speech API url",
             "type": "string"
         },
+        "asr_use_enhanced": {
+            "default": true,
```

Comment:
Think will be correct use `false` value.
That because this required collecting user data audio on Google servers side.
https://cloud.google.com/speech-to-text/docs/enhanced-models

Think we cannot enable this by default.

Reply:
I believe iSpeech does this by default anyway. I could be wrong though.

https://www.ispeech.org/policies.php